src/laboratory/gtselects.py

```diff
@@ -124,7 +124,7 @@ class AvailableContainerLookup(BaseSelect2View):
 
     def get_text_display(self, obj):
         text_display = f"[" + _("Shelf") + f" {obj.shelf.name}] - {obj.object.code} {obj.object.name}"
-        if hasattr(obj.object, 'materialcapacity'):
+        if hasattr(obj.object, 'materialcapacity') and obj.object.is_container:
```

Comment:
This validation shouldn't happen here at all... because if it doesn't have material capacity still returns things and material capacity has nothing to do with the is_container field.  This is_container validation needs to happen in the utils method, because at the end of the day we need that filter also for the serializers and other places, not only for the gtselects.  Same for the next one ContainersForCloningLookup.  

Please move the filter about is_container to the utils methods respectively and test that the serializer is actually also filtering out those that are not containers.

Reply:
Wait, I see you did add it to the utils method (sorry about that), so why is it necessary here? the utils will filter out those that are not containers, so why also add a validation in the get_text_display method?  I do think it is necessary.

Reply:
yeah, but as far as the material has the is container as true then they are displayed, then the **if material capacity** that is already there will consider if they have it or not and add it to the text to display, if not then nothing will be added and the default string without them will be sent... after the migration is applied we can create those with is container as false and they won't be displayed, won't get returned by the utils method and won't get into that get_text_display method, meaning that method really don't care if the is_container is set... am I missing something?